web/src/app/main/AppRoutes.tsx

```diff
@@ -71,6 +71,7 @@ const alertQuery = gql`
 // Allow any component to be used as a route.
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const routes: Record<string, JSXElementConstructor<any>> = {
+  '/': AlertsList,
```

Comment:
I don't think this will ever render; we redirect `/` to `/alerts` before that happens

Reply:
Without this change, you'll see a flicker of "Page does not exist" before the alerts page renders